### PR TITLE
Create new profile

### PR DESCRIPTION
**Pull request by david-kariuki**
Create a new profile for David Kariuki [David Kariuki - Github](https://www.github.com/david-kariuki)

## Check List (Check all the applicable boxes) 

- [X ] My code follows the code style of this project.
- [X ] My change requires changes to the documentation.
- [ X] I have updated the documentation accordingly.
- [ X] All new and existing tests passed.
- [ X] This PR does not contain plagiarized content.
- [ X] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2853"></a>

### CODE REVIEW

**Review by Njong392 — APPROVED** [review]
Looks great! Thank you for contributing :tada:

File: data/david-kariuki.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+         "url":"https://twitter.com/davidkariuki_",
+         "icon":"FaTwitter"
+      },
+      {
+         "name":"Follow me on Instagram",
+         "url":"https://instagram.com/david_kariuki",
+         "icon":"FaInstagram"
+      }
+   ],
+   "tags":[
```
Comment (Njong392) [inline data/david-kariuki.json:29]: You could move the `tags` section here up directly before the `links` part.

**Review by Njong392 — CHANGES_REQUESTED** [review]

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍 

Great collaboration 💯

**Comment by david-kariuki** [comment]
Thank you

**Pull request merged**